source/autocomplete/tests/test.cpp

```diff
@@ -612,7 +612,7 @@ BOOST_AUTO_TEST_CASE(autocomplete_main_stop_area_test) {
     b.sa("Zebre", 0, 0);
 
     b.make();
-    Admin* ad = new Admin;
+    auto* ad = new Admin;
```

Comment:
I am not really a fan of replacing a concrete type by `auto`, as it becomes harder to know what type of object is being used while reading. But I admit that here it doesn't matter much :)
What do you think ?

Reply:
I agree with you but for some case like this one : Admin* ad = new Admin; 
Admin is repeated twice, so using Admin is a good choice. 
Clang-tidy should be smart enough to handle cases where readability is not improved. 
See following documentation about auto keywork https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-auto.html

Reply:
allright, clang is smart :+1: